swap/src/protocol/alice.rs

```diff
@@ -1,30 +1,32 @@
 //! Run an XMR/BTC swap in the role of Alice.
 //! Alice holds XMR and wishes receive BTC.
-use self::{amounts::*, message0::*, message1::*, message2::*, message3::*};
+pub use self::{amounts::*, message0::*, message1::*, message2::*, message3::*, state::*};
```

Comment:
I usually prefer for the `pub use` to be separated so it's clear what does the module re-exports.